esp/scm/ws_dfu.ecm

```diff
@@ -133,6 +133,7 @@ ESPStruct [nil_remove] DFUFileDetail
     [min_ver("1.09")] string UserPermission;
     [min_ver("1.21")] string ContentType;
     [min_ver("1.22")] int64 CompressedFileSize;
+    [min_ver("1.34")] float PercentCompressed;
```

Comment:
I tested yesterday. The result looks correct (I think that was by accident). I will switch it to string.